tests/ui-api-layer-acceptance-tests/graphql/client.go

```diff
@@ -4,36 +4,39 @@ import (
 	"context"
 	"crypto/tls"
 	"net/http"
-	"os"
 	"time"
 
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+
 	"github.com/machinebox/graphql"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
```

Comment:
Please do not introduce a new dependency that is used only in one place

Reply:
done